pom.xml

```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>emissary</groupId>
   <artifactId>emissary</artifactId>
-  <version>7.7.0-SNAPSHOT</version>
+  <version>7.7.0-tika-update-SNAPSHOT</version>
```

Comment:
Remove this change of version

Reply:
i'm working on updating it now. What's there now is very much out of date